js/super_hero.js

```diff
@@ -1 +1,77 @@
-// var SuperHero = ...
+var Superhero = function(name, dmgPoints, hitPoints) {
+	if (dmgPoints <= 0) {
+		throw new Error('Damage cannot be less than 0');
+	}
+	if (hitPoints <= 0) {
+		throw new Error('Health cannot be less than 0');
+	}
+	this.name = name;
+	this.dmgPoints = dmgPoints;
+	this.hitPoints = hitPoints || 100;	
+	this.currentHitPoints = this.hitPoints;
+	console.log(this + ' enters the arena! HEALTH: ' + this.hitPoints + ', POWER: ' + this.dmgPoints);
```

Comment:
You might consider adding a `toString` method for helping to identify the `Superhero` that you are printing out.